packages/suite/src/actions/settings/deviceSettingsActions.ts

```diff
@@ -62,6 +62,39 @@ export const changePin =
         }
     };
 
+export const changeWipeCode =
+    (params: Parameters<typeof TrezorConnect.changeWipeCode>[0] = {}, skipSuccessToast?: boolean) =>
+    async (dispatch: Dispatch, getState: GetState) => {
+        const device = selectDevice(getState());
+
+        if (!device) return;
+
+        const result = await TrezorConnect.changeWipeCode({
+            device: {
+                path: device.path,
+            },
+            ...params,
+        });
+        if (result.success) {
+            if (!skipSuccessToast) {
+                dispatch(notificationsActions.addToast({ type: 'wipe-code-changed' }));
+            }
+        } else if (result.payload.code === 'Failure_PinMismatch') {
```

Comment:
please test mismatch when setting up wipe code on model One. imho it does not return `Failure_PinMismatch` but something like WipeCodeMismatch maybe?

Reply:
could be that this branch (else if) is intended to handle something that will never happen here